internals/secrethub/generate.go

```diff
@@ -282,6 +318,10 @@ type boolValue struct {
 	v *bool
 }
 
+func (iv *boolValue) Type() string {
+	panic("boolValue")
```

Comment:
I think we can drop the bool value and use the bool flag type instead.

Reply:
Indeed, nice spot! Fixed it.